sites/public/pages/finder.tsx

```diff
@@ -89,15 +115,15 @@ const Finder = () => {
             {t("listingFilters.buttonTitleExtended")}
           </div>
           <StepHeader
-            currentStep={currentStep}
+            currentStep={sectionNumber}
             totalSteps={3}
             stepPreposition={t("finder.progress.stepPreposition")}
             stepLabeling={stepLabels}
           ></StepHeader>
         </div>
         <ProgressNav
-          currentPageSection={currentStep}
-          completedSections={currentStep - 1}
+          currentPageSection={sectionNumber}
```

Comment:
Sort of an aside, but it seems like maybe the progress nav doesn't need to keep track of both of these numbers if `completedSections` is always one less than `currentPageSection`?

Reply:
Good point, @emilyjablonski. The only use cases I can think of come up with the disclaimer PR if we wanted to show all steps as completed. It's not currently on the Figma, but I believe I remember @slowbot mentioning that would be preferred if it wasn't too much additional work. The other one would be if we wanted to show skipped steps as empty, but don't think we've ever discussed this case.

Reply:
Update: Was working ahead on the disclaimer piece and setting the currentPageSection and completedSections to the same value results in the final step still showing as active. It only works with the currentPageSection set to one greater than the total number of steps so the completed step value is, once again, one less that the current 😮. So now the _only_ case I can think of is if we ever wanted something like the screenshot below.
<img width="997" alt="image" src="https://user-images.githubusercontent.com/53269332/192658223-102ce97f-e0f8-4c74-a5b9-e4c7484f2b75.png">

Reply:
Sounds great thank you for the details! I think good to leave as is for now.